tests/data/testJson.js

```diff
@@ -211,6 +211,18 @@ const testCases = [
             error: 0,
         },
     },
+    {
+        name: 'typescript : hello world',
+        reqObject: {
+            language: 'typescript',
+            script: 'console.log("hello world");',
+        },
+        expectedResponse: {
+            val: 'hello world\n',
+            status: 200,
+            error: 0,
+        },
+    },
```

Comment:
Hey can you please add a test case here that takes an input and then logs the same on console ?